Units/parser-php.r/run-guest-nested.d/input.php

```diff
@@ -0,0 +1,39 @@
+<html><head><title>X</title>
+<?php // This test input is derived from #2256 submitted by @StephenWall.
+  function draw($x) {
+      echo "$x";
+  }
+?>
+    <meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
+    <title>My Home Page</title>
+    <link rel="stylesheet" href="css/general.css" type="text/css">
+    <style type="text/css"></stylE>
+  </head>
+  <body>
+    <h1>stuff</h1>
+    <a href="/blah">nowhere</a>
+    <?=draw('nowhere')?>
+    <a href="/blech">somewhere</a>
+    <?=draw('somewhere')?>
+  </body>
+</html>
+<script>
+    var f1 = function (n) {
+	return n + <?php /* some value */ ?>;
+    }
+</script>
+<script>
+    var f2 = function (m) {
+	return m + <% /* some value from JSP */ %>;
+    }
+</script>
+<style>
+.blarg {
+    position: relative;
+}
+</style>
+<?php
+  function nothing($x) {
+      ;
+  }
+?>
```

Comment:
Could you put a new line at the end?

Reply:
(If possible, I should put a rule to .editorconfig forcing this.)

Reply:
(You don't have to take any extra action.)

My comment here is wrong.
About input.*, any contents are acceptable.
The new line at the EOF is generally expected.
However, for the test input, there is no common expectation.
We may want to test ctags with various broken inputs.

Feel free to revert the change you had to introduce for responding to my review comment.